src/doc-utils.js

```diff
@@ -126,11 +126,19 @@ function propagateBreaks(doc) {
   );
 }
 
+function getNodeSource (node, text) {
+  return text
```

Comment:
Great but those helper functions are based on the node's range. In our case, the `declare` keyword is not counted in (I tried it):

```js
declare type Foo2 = string; // range: [ 8, 27 ]
```